src/pages/jeonyongmin/Main/components/SubMain/SubMain.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Feed from './Feed/Feed';
+import './SubMain.scss';
+
+class SubMain extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      feedList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/jeonyongmin/feedData.json', {
+      method: 'GET',
+    })
+      .then(result => result.json())
+      .then(data => {
+        this.setState({ feedList: data });
+      });
+  }
+
+  render() {
+    const { feedList } = this.state;
+    return (
+      <div className="feedContainerYM">
+        {feedList.map(element => {
+          return (
+            <Feed
+              key={element.id}
+              number={element.id}
+              name={element.name}
+              feedImg={element.feedImg}
+              profileImg={element.profileImg}
+              comment={element.comment}
+              like={element.like}
+            />
+          );
+        })}
```

Comment:
```suggestion
        {feedList.map(element => (
            <Feed
              key={element.id}
              number={element.id}
              name={element.name}
              feedImg={element.feedImg}
              profileImg={element.profileImg}
              comment={element.comment}
              like={element.like}
            />
          );
        )}
```
arrow function에서 함수 본문에 리턴하는 동작만 수행할경우 중괄호와 return keyword 생략 가능합니다.

Reply:
중괄호 사용시 `return` 필수만 알고 있었는데 감사합니다!!